torchmetrics/metric.py

```diff
@@ -331,6 +392,20 @@ def _apply(self, fn):
                 )
         return this
 
+    @contextmanager
+    def _apply_persistent(
```

Comment:
Where will this be used? Don't see it anywhere

Reply:
Removed 1